stable/sonarqube/templates/deployment.yaml

```diff
@@ -9,6 +9,10 @@ metadata:
     heritage: {{ .Release.Service }}
 spec:
   replicas: {{ .Values.replicaCount }}
+{{- if .Values.deploymentStrategy }}
```

Comment:
@rjkernick - I left it out of the documentation to align it with _replicas_ which is also un-documented. If I was documenting it I'd be recommending the use of `deploymentStrategy: Replace`; however this couldn't be the default as it would cause issues for existing users and require testing the implications. My thought were that adding documentation would be more confusing than non, but I'm happy to add some if you'd like?

Reply:
Done